README.md

```diff
@@ -73,7 +79,7 @@ Options for the CLI are the kebab-case equivalent of the [`@logdna/logger`][] op
 Options:
   -v, --version                   Show version
   -h, --help                      Show usage information
-  -m, --message-key [msg]         The field in the `pino` used as the display line in LogDNA 
+  -m, --message-key [msg]         The field in the `pino` used as the display line in LogDNA
```

Comment:
nit: mind adding the option here as well? the duplication isn't great, I know

Reply:
I forced pushed the fix as you were requesting it :)